packages/wallets/src/features/cashier/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -21,57 +23,66 @@ type TProps = {
     hideWalletDetails: boolean;
 };
 
-const realAccountTabs = [
-    {
-        icon: <LabelPairedPlusMdRegularIcon />,
-        path: 'deposit',
-        text: 'Deposit',
-    },
-    {
-        icon: <LabelPairedMinusMdRegularIcon />,
-        path: 'withdrawal',
-        text: 'Withdraw',
-    },
-    {
-        icon: <LabelPairedArrowUpArrowDownMdRegularIcon />,
-        path: 'account-transfer',
-        text: 'Transfer',
-    },
-    {
-        icon: <LabelPairedSquareListMdRegularIcon />,
-        path: 'transactions',
-        text: 'Transactions',
-    },
-] as const;
+const getRealAccountTabs = (localize: ReturnType<typeof useTranslations>['localize']) => {
+    const realAccountTabs = [
+        {
+            icon: <LabelPairedPlusMdRegularIcon />,
+            path: 'deposit',
+            text: localize('Deposit'),
```

Comment:
hey @lubega-deriv 
Could we do smth like this:  text: `<Localize i18n_default_text="Deposit" />` ?
in that case there is no need to pass `localize` function 🤔

Reply:
Updated